config/environments/production.rb

```diff
@@ -60,18 +60,9 @@
   # Use a different cache store in production
   config.cache_store = :memory_store
 
-  # Enable serving of images, stylesheets, and JavaScripts from an asset server
-  if ENV['ASSET_SYNC_ENABLED'] == 'false'
-     # noop
-  elsif ENV['CLOUDFRONT_HOST']
+  # Enable serving of images, stylesheets, and JavaScripts from Cloudfront
+  if ENV["CLOUDFRONT_HOST"]
```

Comment:
How does that work with using cloudfront if we're not keeping the assets in sync any more? Or is it some other component that's responsible for syncing assets to cloudfront?